dev/devicelab/bin/tasks/technical_debt__cost.dart

```diff
@@ -15,10 +15,12 @@ const double todoCost = 1009.0; // about two average SWE days, in dollars
 const double ignoreCost = 2003.0; // four average SWE days, in dollars
 const double pythonCost = 3001.0; // six average SWE days, in dollars
 const double skipCost = 2473.0; // 20 hours: 5 to fix the issue we're ignoring, 15 to fix the bugs we missed because the test was off
+const double ignoreForFileCost = 2477.0; // similar thinking as skipCost
 const double asDynamicCost = 2003.0; // same as ignoring analyzer warning
 
 final RegExp todoPattern = new RegExp(r'(?://|#) *TODO');
 final RegExp ignorePattern = new RegExp(r'// *ignore:');
+final RegExp ignoreForFilePattern = new RegExp(r'// *ignore_for_file:');
```

Comment:
We may want to be able to whitelist entries.  e.g.

https://github.com/flutter/flutter/blob/980c8a737b7ed211cc86fcf6a84538e71f5c6c25/dev/tools/gen_date_localizations.dart#L83

Reply:
That's the entry that made me add this, actually. I plan to remove it.